hadoop-ozone/client/src/test/java/org/apache/hadoop/ozone/client/rpc/RpcClientTest.java

```diff
@@ -149,7 +149,17 @@
         OZONE_OM_CLIENT_PROTOCOL_VERSION,
         "1.0.1",
         "1.0.1",
-        false);;
+        false),
+    VALID_EXPECTED_TWO_OM_ONE_LOWER_VERSION(
+        OZONE_OM_CLIENT_PROTOCOL_VERSION,
+        "1.0.1",
+        "2.0.1",
+        false),
+    VALID_EXPECTED_TWO_OM_ONE_EMPTY(
+        OZONE_OM_CLIENT_PROTOCOL_VERSION,
+        "1.0.1",
```

Comment:
Thanks for adding.
Can we have  both "", as in this case "" is not compared, as first version is lower code returns false.

Reply:
done